### ISSUE TITLE
SW-1. replace watchdog error information by the HDLC format

### ISSUE THREAD

**Issue by changtengfei** [openwsn-berkeley/openwsn-on-iotlab#1]
Currently watchdog error is recorded as: \nBOARD_RESET!\n
https://github.com/openwsn-berkeley/openwsn-on-iotlab/blob/master/A8/experiment/watchdog.py#L11

It should be replaced by the corresponding opeserial Error output, as this function printed out:
https://github.com/openwsn-berkeley/openwsn-fw/blob/develop_FW-548/drivers/common/openserial.c#L816

**Comment by twatteyne** [comment]
good catch. Maybe we should track issues in the usual tracker for this project, under SW?
